std_msgs/Header header

link_params link1
link_params link2
link_params link3
link_params link4
link_params link5
link_params link6
link_params link7

float64[7] tau_cmd

float64[7] error_q
float64[7] dot_error_q
float64[7] q_cur
float64[7] ddot_q_curr